test/logplex_logs_rest_SUITE.erl

```diff
@@ -57,10 +81,67 @@ v1_redirects_sessions(Config) ->
     Post = binary_to_list(iolist_to_binary([?config(logs, Config),
            "/sessions/"])),
     PostRes = logplex_api_SUITE:post(Post, [{headers, [{"Authorization", BasicAuth}]},
-                                       {http_opts, [{autoredirect, false}]}]),
+                                            {http_opts, [{autoredirect, false}]}]),
     302 = proplists:get_value(status_code, PostRes),
     ok.
 
+post_logline(Config) ->
+    BasicAuth = ?config(auth, Config),
+    Post = binary_to_list(iolist_to_binary([?config(logs, Config),
+           "/logs/"])),
+    Token = ?config(token, Config),
+    meck:expect(logplex_logs_rest, is_authorized,
+                fun(Req, State) ->
+                        State1 = setelement(2, State, Token),
+                        State2 = setelement(3, State1, <<"dont'care">>),
+                        State3 = setelement(4, State2, 3),
```

Comment:
What do these indexes stand for?  I'd like an EOL comment like `% token` and whatever